core/string/ustring.cpp

```diff
@@ -34,12 +34,12 @@
 #include "core/math/color.h"
 #include "core/math/math_funcs.h"
 #include "core/os/memory.h"
+#include "core/string/node_path.h"
 #include "core/string/print_string.h"
 #include "core/string/translation.h"
 #include "core/string/ucaps.h"
 #include "core/variant/variant.h"
 #include "core/version_generated.gen.h"
-
```

Comment:
This separation line was intentional.

Reply:
I wonder why keep getting those removed from time to time when I change includes, I have the feeling QtCreator is doing something odd.